signac/__main__.py

```diff
@@ -537,17 +537,24 @@ def _sig(st):
                 print(MSG_SYNC_STATS.format(stats=stats))
     except SchemaSyncConflict as error:
         _print_err(
-            "WARNING: The detected schemas of the two projects differ! "
-            "Use --force to ignore.")
-        only_in_src = error.schema_src.difference(error.schema_dst)
+            "Synchronizing two projects with different schema requires the -m/--merge option.")
+        diff_src = error.schema_src.difference(error.schema_dst)
+        diff_dst = error.schema_dst.difference(error.schema_src)
+        only_in_dst = diff_dst.difference(diff_src)
+        only_in_src = diff_src.difference(diff_src)
+        diff_value = diff_src.intersection(diff_dst)
         if only_in_src:
             keys_formatted = ('.'.join(k) for k in only_in_src)
             _print_err("Keys found only in the source schema: {}".format(', '.join(keys_formatted)))
-        only_in_dst = error.schema_dst.difference(error.schema_src)
         if only_in_dst:
             keys_formatted = ('.'.join(k) for k in only_in_dst)
             _print_err(
                 "Keys found only in the destination schema: {}".format(', '.join(keys_formatted)))
+        if diff_value:
+            keys_formatted = ('.'.join(k) for k in diff_value)
+            _print_err(
+                "Keys having differnt values in source and destination: {}"
```

Comment:
```suggestion
                "Keys having different values in source and destination: {}"
```

Reply:
> @vishav1771 I don't have push permissions to your repo. Can you commit my suggestion directly? I will merge after that.

Sure.
> 
> Also, in general it is helpful for us if you complete the checklist when you make the pull request. That lets us track how far you've gotten.


Okay will do it  next time :)